packages/landingpage/pages/[lang]/library/components/index.tsx

```diff
@@ -10,28 +10,48 @@ import { LangContext } from '../../../../types/langContext';
 import { Locale_File } from '../../../../translations/types';
 import { merge } from 'lodash';
 import { useStorybookUrlSyncer } from '../../../../utils/hooks/useStorybookUrlSyncer';
+import { useContext, useEffect } from 'react';
+import { UiContext, UiContextType } from '../../../../utils/context/UiContext';
+import Page from '../../../../components/layout/page';
+import useTranslation from '../../../../utils/hooks/useTranslation';
 
 const StoryBookPage: NextPage<void> = () => {
+  const { t } = useTranslation();
+  const { setHideFooter } = useContext(UiContext) as UiContextType;
+
   const initialStorybookUrl = useInitialStorybookUrl();
   useStorybookUrlSyncer();
 
+  // prevent scrolling of body while in storybook
+  useEffect(() => {
+    document.body.style.overflow = 'clip';
+    setHideFooter(true);
+
+    return () => {
+      document.body.style.overflow = 'initial';
+      setHideFooter(false);
+    };
+  }, []);
```

Comment:
Why is this necessary? I couldn't tell a difference between the scroll behavior on master vs this branch

Reply:
The problem can be reproduced in the most current deployment of our landingpage:
![elements-scroll-landingpage](https://github.com/inovex/elements/assets/22963121/a458f819-3b11-4474-ac61-f877f67403bd)

The problem is that the whole page will be scrollable as soon as you reach the max scroll distance of the page inside the iframe. That makes it kinda awkward to use as you have to be careful not to "over scroll" the storybook page.